megago/resource_data/go_freq_uniprot.json

```diff
@@ -0,0 +1 @@
32550": 14590, "GO:0032551": 0, "GO:0032552": 27, "GO:0032553": 104636, "GO:0032554": 23, "GO:0032555": 102395, "GO:0032556": 5, "GO:0032557": 20, "GO:0032558": 16, "GO:0032559": 88724, "GO:0032560": 8, "GO:0032561": 14762, "GO:0032562": 0, "GO:0032563": 0, "GO:0032564": 14, "GO:0032565": 0, "GO:0032566": 0, "GO:0032567": 8, "GO:0032570": 189, "GO:0032571": 51, "GO:0032572": 1, "GO:0032573": 0, "GO:0032574": 14, "GO:0032576": 1, "GO:0032577": 1, "GO:0032578": 9, "GO:0032579": 1, "GO:0032580": 401, "GO:0032581": 20, "GO:0032584": 49, "GO:0032585": 114, "GO:0032586": 17, "GO:0032587": 316, "GO:0032588": 205, "GO:0032589": 311, "GO:0032590": 193, "GO:0032591": 49, "GO:0032592": 663, "GO:0032593": 41, "GO:0032594": 34, "GO:0032595": 3, "GO:0032596": 15, "GO:0032597": 3, "GO:0032598": 0, "GO:0032599": 3, "GO:0032600": 3, "GO:0032601": 0, "GO:0032602": 16, "GO:0032603": 0, "GO:0032604": 1, "GO:0032605": 0, "GO:0032606": 27, "GO:0032607": 8, "GO:0032608": 14, "GO:0032609": 27, "GO:0032610": 5, "GO:0032611": 25, "GO:0032612": 25, "GO:0032613": 12, "GO:0032614": 0, "GO:0032615": 26, "GO:0032616": 1, "GO:0032617": 0, "GO:0032618": 2, "GO:0032619": 0, "GO:0032620": 7, "GO:0032621": 4, "GO:0032622": 0, "GO:0032623": 15, "GO:0032624": 0, "GO:0032625": 0, "GO:0032626": 0, "GO:0032627": 0, "GO:0032628": 0, "GO:0032629": 0, "GO:0032630": 0, "GO:0032631": 0, "GO:0032632": 4, "GO:0032633": 3, "GO:0032634": 1, "GO:0032635": 39, "GO:0032636": 0, "GO:0032637": 5, "GO:0032638": 0, "GO:0032639": 0, "GO:0032640": 18, "GO:0032641": 0, "GO:0032642": 294, "GO:0032643": 6, "GO:0032644": 5, "GO:0032645": 48, "GO:0032646": 6, "GO:0032647": 92, "GO:0032648": 144, "GO:0032649": 385, "GO:0032650": 28, "GO:0032651": 271, "GO:0032652": 328, "GO:0032653": 167, "GO:0032654": 0, "GO:0032655": 199, "GO:0032656": 59, "GO:0032657": 0, "GO:0032658": 2, "GO:0032659": 0, "GO:0032660": 144, "GO:0032661": 20, "GO:0032662": 0, "GO:0032663": 195, "GO:0032664": 0, "GO:0032665": 0, "GO:0032666": 0, "GO:0032667": 45, "GO:0032668": 0, "GO:0032669": 0, "GO:0032670": 0, "GO:0032671": 0, "GO:0032672": 8, "GO:0032673": 112, "GO:0032674": 56, "GO:0032675": 523, "GO:0032676": 0, "GO:0032677": 239, "GO:0032678": 0, "GO:0032679": 5, "GO:0032680": 540, "GO:0032681": 1, "GO:0032682": 58, "GO:0032683": 0, "GO:0032684": 0, "GO:0032685": 2, "GO:0032686": 6, "GO:0032687": 14, "GO:0032688": 39, "GO:0032689": 118, "GO:0032690": 8, "GO:0032691": 95, "GO:0032692": 117, "GO:0032693": 58, "GO:0032694": 0, "GO:0032695": 52, "GO:0032696": 15, "GO:0032697": 0, "GO:0032698": 0, "GO:0032699": 0, "GO:0032700": 73, "GO:0032701": 4, "GO:0032702": 0, "GO:0032703": 76, "GO:0032704": 0, "GO:0032705": 0, "GO:0032706": 0, "GO:0032707": 7, "GO:0032708": 0, "GO:0032709": 0, "GO:0032710": 0, "GO:0032711": 0, "GO:0032712": 3, "GO:0032713": 16, "GO:0032714": 15, "GO:0032715": 173, "GO:0032716": 0, "GO:0032717": 52, "GO:0032718": 0, "GO:0032719": 0, "GO:0032720": 196, "GO:0032721": 0, "GO:0032722": 225, "GO:0032723": 6, "GO:0032724": 5
```

Comment:
For reproducibility, we should note the version of the uniprot DB that this json is based on. Can we include an additional property holding the database version, e.g. as a simple date stamp (YYYY-MM-DD)?